PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/springcloud/SpringCloudAppPropertyView.java

```diff
@@ -239,6 +245,8 @@ private void onLoadSpringCloudApp(String subscription, String resourceGroup, Str
         lblCreateTime.setText(LOADING);
         lblAppName.setText(LOADING);
 
+        btnIsPublic.setText(LOADING);
```

Comment:
why set button as `loading`, we may disable it before initialization?

Reply:
We don't know whether it is public or not at initial stage